OpenRA.Mods.Common/Widgets/Logic/Lobby/LobbyUtils.cs

```diff
@@ -517,5 +517,35 @@ public static string GetExternalIP(int clientIndex, OrderManager orderManager)
 
 			return address;
 		}
+
+		public static void SetupChatPanel(ContainerWidget template, Color c, string from, string text)
```

Comment:
This name implies that it is setting up the whole chat panel, rather than just one line.  Please rename to `SetupChatLine`.